src/phoenix/db/insertion/span.py

```diff
@@ -38,89 +38,62 @@ async def insert_span(
 
     project_session: Optional[models.ProjectSession] = None
     session_id = get_attribute_value(span.attributes, SpanAttributes.SESSION_ID)
+    session_user = get_attribute_value(span.attributes, SpanAttributes.USER_ID)
     if session_id is not None and (not isinstance(session_id, str) or session_id.strip()):
         session_id = str(session_id).strip()
         assert isinstance(session_id, str)
+        session_user = str(session_user).strip()
+        assert isinstance(session_user, str)
```

Comment:
we're not guaranteed to have a user with a session right? i think we need to check if it's not None here